pymatgen/io/ase.py

```diff
@@ -38,18 +40,41 @@
 __date__ = "Mar 8, 2012"
 
 
+class PMGAtoms(Atoms, MSONable):
```

Comment:
one thing to consider is how this affects any future plans we might hatch to develop a 1st class `Atoms` object in `pymatgen`. there were times where i thought a single class to handle periodic and non-periodic systems would be nice. just to keep that option open, maybe it would be better to reserve the name `PmgAtoms` (even if we later just call it `Atoms` or similar) to avoid future confusion and name the new class in this PR `AseAtoms` instead?

Reply:
Personally, I think `AseAtoms` is somewhat misleading, as it is not the ASE `Atoms` object. I'm open to other suggestions though.

Sidenote: We might want to consider `MSONableAtoms` as another option to make the purpose clearer, although that doesn't directly address your concern. Also, `PmgAtoms` or `PMGAtoms`?

Reply:
i don't know if i'd consider it misleading given it's in every way identical except for some added serialization behavior. after all, according to Python 

```py
isinstance(pymatgen.io.ase.AseAtoms(), ase.atoms.Atoms)
>>> True
```

that said, i like `MSONableAtoms` even better as, like you said, it communicates intent.

i find pascal case `PmgAtoms` more readable than `PMGAtoms` but i'm sure that's debatable 😄